examples/flutter_gallery/test_driver/transitions_perf_test.dart

```diff
@@ -201,6 +201,10 @@ void main([List<String> args = const <String>[]]) {
 
     test('all demos', () async {
 
+      // Let it fully start to ensure that we only measure the kProfiledDemos
+      // See https://github.com/flutter/flutter/pull/20400 for a similar change.
```

Comment:
not sure what the reader gains from a link to a PR. Probably best to link to an issue.

That said, we should really remove both of these delays and replace them with more reasonable delays that actually wait for a specific time, e.g. wait until CPU usage is below 1% or whatever.

Also if we find we need this with multiple tests, we should consider moving this into the driver framework somewhere so we only have the hack in one place instead of duplicating it everywhere.